locust/test/test_web.py

```diff
@@ -52,6 +52,19 @@ def test_web_ui_no_runner(self):
     
     def test_index(self):
         self.assertEqual(200, requests.get("http://127.0.0.1:%i/" % self.web_port).status_code)
+
+    def test_index_with_options(self):
+        self.environment.parsed_options = parse_options(["-r", "10"])
+        self.assertEqual(200, requests.get("http://127.0.0.1:%i/" % self.web_port).status_code)
```

Comment:
We should check that the values has been set in the response HTML as well. Either just check for the whole `<input ...>` string, or with a simple regex, or perhaps even using [pyquery](https://pyquery.readthedocs.io/en/latest/) (which we would have to add to the `tests_require` dependencies in setup.py).